### PR TITLE
CPDP-839 Extend unit test coverage for task list page.

### PR DESCRIPTION
**Pull request by rszmacinski**

### CODE REVIEW

File: test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/TaskListControllerSpec.scala [hunk 1]
```diff
@@ -82,15 +89,120 @@ class TaskListControllerSpec
...
+                )
+              )
+            )
+          }
+
+          val result = performAction()
+          status(result) shouldBe OK
+
+          val doc: Document = parse(contentAsString(result))
+          doc.title shouldBe "Report and pay Capital Gains Tax on UK property - GOV.UK"
```
Comment (AndyHWChung) [inline test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/TaskListControllerSpec.scala]: can we assert against `messagefromMessageKey("service.title")` here?

File: test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/TaskListControllerSpec.scala [hunk 2]
```diff
@@ -82,15 +89,120 @@ class TaskListControllerSpec
...
+                  )
+                )
+              )
+            )
+          }
+
+          val result = performAction()
+          status(result) shouldBe OK
 
+          val doc: Document = parse(contentAsString(result))
```
Comment (AndyHWChung) [inline test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/TaskListControllerSpec.scala]: could we assert the page title as well? It gives us a bit more confidence that we're on the right page

File: test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/TaskListControllerSpec.scala [hunk 3]
```diff
@@ -82,15 +89,120 @@ class TaskListControllerSpec
...
+          doc.select("a#saveAndComeBackLater").attr("href") shouldBe "/capital-gains-tax-uk-property/account-home"
```
Comment (AndyHWChung) [inline test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/TaskListControllerSpec.scala]: you should be able to get this url from `controllers.accounts.homepage.HomepageController.homepage()` or something like that

File: project/AppDependencies.scala [hunk 4]
```diff
@@ -17,7 +17,7 @@ object AppDependencies {
 
   val test = Seq(
     "org.scalatest"              %% "scalatest"                 % "3.0.8"          % "test",
-    "org.jsoup"                  % "jsoup"                      % "1.10.2"         % "test",
+    "org.jsoup"                  % "jsoup"                      % "1.8.3"         % "test",
```
Comment (AndyHWChung) [inline project/AppDependencies.scala]: is there a reason we can't jump to the latest?

Comment (rszmacinski) [reply project/AppDependencies.scala]: done, it was some kind of mistake :) Good spot, thanks

File: test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/TaskListControllerSpec.scala (hunk 3 above)
Comment (rszmacinski) [reply test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/TaskListControllerSpec.scala]: done

File: test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/TaskListControllerSpec.scala (hunk 2 above)
Comment (rszmacinski) [reply test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/TaskListControllerSpec.scala]: I will discuss it personally with you, don't think we need nor want to check that.

Comment (rszmacinski) [reply test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/TaskListControllerSpec.scala]: done

File: test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/TaskListControllerSpec.scala (hunk 1 above)
Comment (rszmacinski) [reply test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/TaskListControllerSpec.scala]: done

**Comment by AndyHWChung** [comment]
LGTM

**Pull request merged**